### DISCUSSION TITLE
which rpi4 image? factory vs sysupgrade

### DISCUSSION THREAD

**Discussion by nmz787** [SmoothWAN/SmoothWAN#46] [General & Help]
Maybe I am too out-of-touch with raspi terminology, but which image do I download from the releases? Is that listed on the instructions page (where is that??)? Do I need to use one first, then the other?

I already use linux, can I just `dd if=~/Download/the_image of=/dev/mmcblk0` those image files to a microSD card (the drive itself, not a partition)? Sometimes I've had issues with my laptop's SD card slot, and need to use a USB adapter instead (which is just slower, and probably irrelevant to this discussion).

P.S. I've got a raspi 400 kit in the mail, should be here in 3 days.

**Comment by bt61** [comment]
"Tutorial" section:

> Tutorial: Prepare an SD card for the Raspberry Pi 4
> 1. Download the file "smoothwan-0.99.6-bcm27xx-bcm2711-rpi-4-ext4-factory.img.gz" from below.
> 2. Download and install balenaEtcher from [balena.io](https://www.balena.io/etcher/)

https://github.com/TalalMash/SmoothWAN/releases/tag/0.99.6-hotfix

**Reply by nmz787** [reply]
I had to search the first line of your quote to find where you actually mean... and I could only find it as an expandable section of one of the old release notes! That info definitely needs copied to the wiki.

Also, looks like `balena` is equivalent to `dd` plus a re-read verification step:
https://github.com/balena-io/etcher/issues/3015

Thanks!

**Reply by TalalMash** [reply]
Copied to Wiki.
Thanks.
